client/app/components/Input/__tests__/InputTag.spec.jsx

```diff
@@ -8,35 +8,35 @@ describe('InputTag', () => {
   it('type in value and select it', () => {
     const wrapper = mount(component);
     const id = `#${InputMocks.inputTagProps.checkboxes[1].id}`;
-    const value = { value: 'Two' };
+    // const value = { value: 'Two' };
     // Hack: Looks like react-autocomplete is looking for target, but we should use currentTarget
-    wrapper.find('.tagAutocomplete').prop('onChange')({
-      target: value,
-      currentTarget: value,
-    });
+    // wrapper.find('.tagAutocomplete').prop('onChange')({
+    //   target: value,
+    //   currentTarget: value,
+    // });
```

Comment:
Why is this commented out? We should still be testing for this.

Reply:
Hey Julia, I think I've figured what I need to do here. I am still working on it but home to put up for this test soon enough.

Reply:
So I reckon I have the answer to it, the test I have instead of the commented out test is:

`describe('InputTag', () => {
  it('type in value and select it', () => {
    const wrapper = mount(component);
    const id = `#${InputMocks.inputTagProps.checkboxes[1].id}`;
    const value = { value: 'Two' };
    expect(wrapper.find('.tagMenu').exists()).toEqual(true);
    wrapper.find('.tagAutocomplete').simulate('change', { target: value });
    wrapper.find('.tagAutocomplete').simulate('focus');
    expect(wrapper.find(id).exists()).toEqual(false);
    wrapper
      .find('.tagLabel')
      .at(0)
      .simulate('click');
    expect(wrapper.find(id).exists()).toEqual(true);
  });

  it('un-check existing value and retype and select it', () => {
    const wrapper = mount(component);
    const id = `input#${InputMocks.inputTagProps.checkboxes[0].id}`;
    const value = { value: 'One' };
    expect(wrapper.find(id).exists()).toEqual(true);
    wrapper.find(id).prop('onChange')({ currentTarget: { checked: false } });
    wrapper.update();
    expect(wrapper.find(id).exists()).toEqual(false);
    wrapper.find('.tagAutocomplete').simulate('change', { target: value });
    wrapper.find('.tagAutocomplete').simulate('focus');
    expect(wrapper.find('.tagMenu').exists()).toEqual(true);
    expect(wrapper.find(id).exists()).toEqual(false);
    wrapper
      .find('.tagLabel')
      .at(0)
      .simulate('click');
    expect(wrapper.find(id).exists()).toEqual(true);
  });
});`

Reply:
I think your proposed change should be fine! Thanks for elaborating and yay for figuring it out.